pkgs/top-level/all-packages.nix

```diff
@@ -24106,7 +24106,7 @@ in
     onlyLibVLC = true;
   };
 
-  vmpk = callPackage ../applications/audio/vmpk { };
+  vmpk = qt5.callPackage ../applications/audio/vmpk { inherit (libsForQt5) drumstick; };
```

Comment:
Please use libsForQt5.callPackage to provide drumstick (`vmpk = libsForQt5.callPackage ../applications/audio/vmpk { };`).